bucket/mremoteng-portable.json

```diff
@@ -0,0 +1,39 @@
+        "if (!(Test-Path \"$persist_dir\\mRemoteNG.log\"  )) { New-Item \"$dir\\mRemoteNG.log\" -ItemType File | Out-Null }",
+        "if (!(Test-Path \"$persist_dir\\extApps.xml\"    )) { New-Item \"$dir\\extApps.xml\"   -ItemType File -Value \"<?xml version='1.0' encoding='utf-8'?>\"  | Out-Null }",
+        "if (!(Test-Path \"$persist_dir\\pnlLayout.xml\"  )) { New-Item \"$dir\\pnlLayout.xml\" -ItemType File -Value \"<?xml version='1.0' encoding='utf-8'?><DockPanel FormatVersion='1.0' DockLeftPortion='225.8' DockRightPortion='225.8' DockTopPortion='130.25' DockBottomPortion='130.25' ActiveDocumentPane='-1' ActivePane='0'>  <Contents Count='3'>    <Content ID='0' PersistString='mRemoteNG.UI.Window.ConnectionTreeWindow' AutoHidePortion='225.8' IsHidden='False' IsFloat='False' />    <Content ID='1' PersistString='mRemoteNG.UI.Window.ConfigWindow' AutoHidePortion='225.8' IsHidden='False' IsFloat='False' />    <Content ID='2' PersistString='mRemoteNG.UI.Window.ErrorAndInfoWindow' AutoHidePortion='130.25' IsHidden='False' IsFloat='False' />  </Contents>  <Panes Count='3'>    <Pane ID='0' DockState='DockLeft' ActiveContent='0'>      <Contents Count='1'>        <Content ID='0' RefID='0' />      </Contents>    </Pane>    <Pane ID='1' DockState='DockLeft' ActiveContent='1'>      <Contents Count='1'>        <Content ID='0' RefID='1' />      </Contents>    </Pane>    <Pane ID='2' DockState='DockBottomAutoHide' ActiveContent='2'>      <Contents Count='1'>        <Content ID='0' RefID='2' />      </Contents>    </Pane>  </Panes>  <DockWindows>    <DockWindow ID='0' DockState='Document' ZOrderIndex='1'>      <NestedPanes Count='0' />    </DockWindow>    <DockWindow ID='1' DockState='DockLeft' ZOrderIndex='2'>      <NestedPanes Count='2'>        <Pane ID='0' RefID='0' PrevPane='-1' Alignment='Bottom' Proportion='0.5' />        <Pane ID='1' RefID='1' PrevPane='0' Alignment='Bottom' Proportion='0.5' />      </NestedPanes>    </DockWindow>    <DockWindow ID='2' DockState='DockRight' ZOrderIndex='3'>      <NestedPanes Count='0' />    </DockWindow>    <DockWindow ID='3' DockState='DockTop' ZOrderIndex='4'>      <NestedPanes Count='0' />    </DockWindow>    <DockWindow ID='4' DockState='DockBottom' ZOrderIndex='5'>      <NestedPanes Count='1'>        <Pane ID='0' RefID='2' PrevPane='-1' Alignment='Right' Proportion='0.5' />      </NestedPanes>    </DockWindow>  </DockWindows>  <FloatWindows Count='0' /></DockPanel>\"  | Out-Null }"
+    ],
+    "persist": [
+        "mRemoteNG.settings",
+        "mRemoteNG.log",
+        "extApps.xml",
+        "pnlLayout.xml"
+    ],
+    "notes": [
+        "Default connections file is not persisted! Use a custom connection file insted.",
+        "On first run, you can load the default panel layout by using 'View'->'Reset Layout'."
+    ],
+    "checkver": {
+        "github": "https://github.com/mRemoteNG/mRemoteNG",
+        "re": "/releases/download/(?<short>[^/]+)/mRemoteNG-Portable-(?<version>[\\d.]+).zip"
```

Comment:
```suggestion
        "regex": "mRemoteNG-Portable-([\\d.]+)\\.zip"
```

Reply:
I copied what the original one had... I agree this looks better.